src/plays/pokemon-stats/PokemonStats.js

```diff
@@ -0,0 +1,70 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import { useState } from "react";
+import "./styles.css";
+import axios from "axios";
+import Pokemoncard from "./components/Pokemoncard";
+import Search from "./components/Search";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+// WARNING: Do not change the entry componenet name
+function PokemonStats(props) {
+  // Your Code Start below.
+  const [pokemon, setpokemon] = useState();
+  const [name, setname] = useState("");
+  const fetchPokemonDetails = (e) => {
+    e.preventDefault();
+    axios
+      .get(`https://pokeapi.co/api/v2/pokemon/${name.toLowerCase()}`)
+      .then((response) => {
+        setpokemon(response.data);
+      })
+      .catch((err) => {
+        toast.error("Sorry, Unable to fetch", {
```

Comment:
Give a better comment like, the Pokemon you are searching is not found, try another.